stories/table.stories.tsx

```diff
@@ -20,6 +23,12 @@ storiesOf('Table', module)
         status: 'married',
         admin: true,
         date: new Date().toString(),
+        editOnClick: () => {
+          Toast('success', 'Edit button clicked!!', 'Success')
+        },
+        deleteOnClick: () => {
+          Toast('success', 'Delete button clicked!!', 'Success')
+        },
```

Comment:
This code is better suited for the columns definition, so you don't have to redeclare it on every patient (and also mixes data and logic)